challenge.js

```diff
@@ -9,7 +9,7 @@
  */
 
 // ANSWER
-
+// Use the inspector in the web browser to find the id of the element you want to change. Use the console to alter the style sheet of that element.
```

Comment:
Not quite. We were looking for how to add a new element to the page, not how to re-style an existing element.